lib/foreman/renderer/configuration.rb

```diff
@@ -42,7 +42,9 @@ class Configuration
         :number_with_delimiter,
         :number_with_precision,
         :number_to_human_size,
-        :gem_version_compare
+        :gem_version_compare,
+        :transpile_coreos_linux_config,
+        :transpile_fedora_coreos_config
```

Comment:
```suggestion
        :transpile_fedora_coreos_config,
```

Reply:
lib/foreman/renderer/configuration.rb:47:40: C: Style/TrailingCommaInArrayLiteral: Avoid comma after the last item of an array, unless each item is on its own line.